vault/testing.go

```diff
@@ -1562,6 +1565,12 @@ func NewTestCluster(t testing.TB, base *CoreConfig, opts *TestClusterOptions) *T
 		c.SharedConfig = &configutil.SharedConfig{LogFormat: logging.UnspecifiedFormat.String()}
 		coreConfig.RawConfig = c
 	}
+	if opts.EnablePostUnsealTrace {
```

Comment:
rather than adding these as options, can retrieve them from the the `base` config? We already set  `coreConfig.RawConfig = base.RawConfig` further down, so as long as we instantiate the `NewTestCluster` as:

```
NewTestCluster(t, &CoreConfig{
  RawConfig: &server.Config{PostUnsealTraceDir: "value", EnablePostUnsealTrace: true},
}, nil)
```
it should work

Reply:
oh good catch, somehow I thought that using the base would overwrite all other settings computed in the function, not sure how I arrived at that conclusion :sweat_smile: